.github/workflows/ci.yml

```diff
@@ -136,9 +134,8 @@ jobs:
           .github/workflows/run.sh make ${{ matrix.target }}
 
   race:
-    runs-on: ubuntu-24.04-8core
+    runs-on: syz-env-x86
     timeout-minutes: 15 # Q1 2024 experiments may affect timeout, let's relax it
```

Comment:
Out of curiosity: what are Q1 2024 experiments?

Reply:
Uffff..... I don't remember at all.....